crates/turborepo-lib/src/commands/generate.rs

```diff
@@ -2,22 +2,31 @@ use std::process::{Command, Stdio};
 
 use anyhow::Result;
 use tracing::debug;
+use which::which;
 
 use crate::{
     child::spawn_child,
     cli::{GenerateCommand, GeneratorCustomArgs},
 };
 
 fn verify_requirements() -> Result<()> {
-    let output = Command::new("npx")
-        .arg("--version")
-        .stdout(Stdio::null())
-        .stderr(Stdio::null())
-        .status();
+    // find npx path
+    let npx_path = which("npx");
```

Comment:
Minor thing, I think we can make use of [`context`](https://docs.rs/anyhow/latest/anyhow/trait.Context.html#example) here and still expose the underlying error and reduce nesting by removing the `match npx_path` statement:
```suggestion
    use anyhow::Context;
    let npx_path = which("npx").context("Unable to run generate - missing requirements (npx)")?;
```

Reply:
Good call, updated!